modules/UI/util/MessageHandler.js

```diff
@@ -453,28 +455,19 @@ const messageHandler = {
     },
 
     /**
-     * Shows a dialog prompting the user to send an error report.
-     *
-     * @param titleKey the title of the message
-     * @param msgKey the text of the message
-     * @param error the error that is being reported
+     *  Shows an error dialog to the user.
+     * @param {string} titleKey - The title of the message.
+     * @param {string} msgKey - The text of the message.
      */
-    openReportDialog(titleKey, msgKey, error) {
-        this.openMessageDialog(titleKey, msgKey);
-        logger.log(error);
-
-        // FIXME send the error to the server
+    showError(props) {
+        APP.store.dispatch(showErrorNotification(props));
     },
 
-    /**
-     *  Shows an error dialog to the user.
-     * @param titleKey the title of the message.
-     * @param msgKey the text of the message.
-     */
-    showError(titleKey = 'dialog.oops', msgKey = 'dialog.defaultError') {
-        messageHandler.openMessageDialog(titleKey, msgKey);
+    showWarning(props) {
```

Comment:
Probably should add docs for this. I don't know why I didn't... If you rather I fix my own little messes just send me a message and I will.